README.md

```diff
@@ -80,13 +81,80 @@ In the end the PCB will not be visible from the outside, powered from the inside
+        Type string : fw_version     "Firmware Version" [ stateTopic="devices/senseo/$fw/version"]
+        Type string : nodes          "Nodes"            [ stateTopic="devices/senseo/$nodes"]
+        Type string : implementation "Implementation"   [ stateTopic="devices/senseo/$implementation"]
+        Type string : interval       "Update interval"  [ stateTopic="devices/senseo/$stats/interval"]
+    }
+}
+```
+
+### SenseoWifi.items
+[expire binding](https://www.openhab.org/addons/bindings/expire1/) recommended for debug-value
+```
+String KU_Senseo_Debug         "Debug [%s]"                          {channel="mqtt:topic:SenseoWiFi:Debug", expire="10s"}
+String KU_Senseo_OpState       "Zustand [MAP(senseo-wifi.map):%s]"   {channel="mqtt:topic:SenseoWiFi:OpState"}
+Switch KU_Senseo_OnOff         "OnOff"                               {channel="mqtt:topic:SenseoWiFi:OnOff"}
+String KU_Senseo_Brew          "Brew"                                {channel="mqtt:topic:SenseoWiFi:Brew"}
+Number KU_Senseo_BrewedSize    "BrewedSize"                          {channel="mqtt:topic:SenseoWiFi:BrewedSize"}
+String KU_Senseo_OutOfWater    "OutOfWater"                          {channel="mqtt:topic:SenseoWiFi:OutOfWater" [profile="transform:MAP", function="senseo-wifi.map"]}
+String KU_Senseo_Recipe        "Recipe"                              {channel="mqtt:topic:SenseoWiFi:Recipe"}
+String KU_Senseo_CupAvailable  "Tasse vorh."                         {channel="mqtt:topic:SenseoWiFi:CupAvailable" [profile="transform:MAP", function="senseo-wifi.map"]}
+String KU_Senseo_CupFull       "Tasse voll"                          {channel="mqtt:topic:SenseoWiFi:CupFull" [profile="transform:MAP", function="senseo-wifi.map"]}
+String KU_Senseo_Online        "Online"                              {channel="mqtt:topic:SenseoWiFi:Online"}
+```
+
+### SenseoWifi.sitemap
+```
+sitemap testing label="Senseo-Tests"
+{
+    Frame label="Status" {
+        //Text item=KU_Senseo_Unreach label="Fehler [Maschine nicht verfügbar!]" visibility=[KU_Senseo_Online == "false"] valuecolor=["true"="red"]
+        Text item=KU_Senseo_OpState label="Aktueller Zustand"
+        Default item=KU_Senseo_Debug      label="Debug" visibility=[KU_Senseo_Debug != UNDEF] valuecolor=["red"]
+        Text item=KU_Senseo_OutOfWater label="Wassertank [Leer]" visibility=[KU_Senseo_OpState == SENSEO_NOWATER] valuecolor=[ON="maroon"] icon="water"
+        Switch item=KU_Senseo_CupAvailable label="Tasse vorhanden" icon="coffee_cup"
+        Switch item=KU_Senseo_CupFull label="Tasse voll" visibility=[KU_Senseo_CupAvailable == ON] icon="coffee_cup_hot"
+    }
+    Frame label="Steuerung" {
+        Switch item=KU_Senseo_OnOff label="Ein-/Ausschalten"
+        Switch item=KU_Senseo_Brew label="Kaffee Kochen" mappings=[1cup="☕", 2cup="☕☕"]
+    }
+}
+```
+
 Further details outstanding. Don't hesitate to open a support issue!
 
 Happy Hacking!
 
 ----
 
-#### Disclaimer and Legal
+# Disclaimer and Legal
```

Comment:
Why? :)

Reply:
Correct me if I'm wrong:
One # is a main-chapter, more of # are sub-chapters for previous main-chapter.
I wanted the disclaimer to be a main-chapter and not only a sub-chapter of `openHAB Configuration Example`

Reply:
Ah I see. Technically correct but my reason for `####` was quite simply to receive a certain font size ;)